LocalyticsXamarin/LocalyticsXamarin.Common/ILocalytics.cs

```diff
@@ -528,8 +521,6 @@ public interface ILocalytics
         /// Inboxs campaigns list.
         /// </summary>
         /// <see cref="DisplayableInboxCampaigns"/>
-        [Obsolete("InboxCampaigns is deprecated, please use DisplayableInboxCampaigns instead.")]
```

Comment:
I don;t think you deleted enough here - I think you need to delete the comment above as well?